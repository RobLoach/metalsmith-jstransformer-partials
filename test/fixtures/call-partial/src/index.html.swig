---
title: Hello, World!
---
<h1>{{ title }}</h1>

<div class="partial-wrapper">
{{ partial('card', {name:'Updated Name!'})|safe }}
</div>

<div class="author-wrapper">
{{ partial('author', {name:'Rob', lastname:'Loach'})|safe }}
</div>

<div class="partial-wrapper">
{{ partial('card')|safe }}
</div>
