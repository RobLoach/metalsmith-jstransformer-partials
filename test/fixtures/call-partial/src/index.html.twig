---
title: Hello, World!
---
<h1>{{ title }}</h1>

<div class="partial-wrapper">
{{ partial('card', {name:'Updated Name!'})|raw }}
</div>

<div class="author-wrapper">
{{ partial('author', {name:'Rob', lastname:'Loach'})|raw }}
</div>

<div class="partial-wrapper">
{{ partial('card')|raw }}
</div>
