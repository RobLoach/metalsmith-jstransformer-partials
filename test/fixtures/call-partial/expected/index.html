<h1>Hello, World!</h1>

<div class="partial-wrapper">


<div class="card">
  <h2>This is a call-partial card!</h2>
  <h3>Name: Updated Name!</h3>
  <h3>Last Name: Default Last Name</h3>
</div>

</div>

<div class="author-wrapper">


<div class="author">
  <h2>This is an author!</h2>
  <h3>Name: Rob</h3>
  <h3>Last Name: Loach</h3>
</div>

</div>

<div class="partial-wrapper">


<div class="card">
  <h2>This is a call-partial card!</h2>
  <h3>Name: Default Name</h3>
  <h3>Last Name: Default Last Name</h3>
</div>

</div>
